app/models/audit/event/api_key.rb

```diff
@@ -0,0 +1,91 @@
+module Audit
+  module Event
+    class ApiKey
+
+      def initialize(auth_role:, subject_role:, success:, error_message: nil)
```

Comment:
The names from the other class were more immediately clear to me: `role_to_rotate authenticated_role`

Reply:
So you think we should use those here as well? 

I was trying to adapt the names to represent the domain of audit, and how they'll manifest in the structured data, but I don't think I have any issue with these names instead.

Reply:
Ah yeah, I still do prefer using the other names too.

I agree with your point about adapting names for their domain, but I'd see the boundary starting only once you call `structured_data`.  That is, the structured_data method converts the human friendly names we created the event with into the jargon-y audit terms.

Reply:
That makes sense, thanks!